README.md

```diff
@@ -122,6 +122,13 @@ To do
 * Improved printing and string input/output
 * IPython hooks (TeX pretty-printing etc.)
 
+CHANGELOG
+-------------
+
+0.5.0
+
+- gh-63: The `roots` method of `arb_poly`, and `nmod_poly` is no longer supported. Use `acb_roots(p).roots()` to get the old behaviour of returning the roots as `acb`. Note that the `roots` method of `fmpz_poly` and `fmpq_poly` currently returns the complex roots of the polynomial.
```

Comment:
```suggestion
- gh-63: The `roots` method of `arb_poly`, and `nmod_poly` is no longer supported. Use `acb_roots(p).roots()` to get the old behaviour of returning the roots as `acb`. Note that the `roots` method of `fmpz_poly` and `fmpq_poly` previously returned the complex roots of the polynomial.
```

Reply:
I'm a bit confused here.

Why are the changes to `arb_poly` being described differently to `fmpz_poly`?

Is the roots method not being removed from all of them except `acb_poly`?

Reply:
They still do. Although I've removed support from the very base class, the fmpz_poly has its own roots method which fmpq_poly inherits. One option is of course to delete the method here as well.

Reply:
Oh, I see.

Okay then this is fine for now.

Reply:
In the future I would like integer roots from fmpz and rational roots from fmpq, but this can wait until we link things up with Flint3 and the genetics I hope. For now this removal allowed the most simple refactoring to start.